code/modules/paperwork/contract.dm

```diff
@@ -45,11 +45,6 @@
 		to_chat(M, "<span class='boldnotice'>You feel that your soul has returned to its rightful owner, Nanotrasen.</span>")
 		M.return_soul()
 	else
-		if(ishuman(M))
-			var/mob/living/carbon/human/N = M
-			if(!istype(N.head, /obj/item/clothing/head/helmet))
-				N.adjustBrainLoss(10)
-				to_chat(N, "<span class='danger'>You feel dumber.</span>")
```

Comment:
So part of the reason for this brainloss is to prevent someone from selling their soul and recovering it repeatedly with their own employment contract.  Eventually the brainloss would get so bad that you would be unable to sell your soul.

Reply:
I replaced the brainloss check with a DUMB check anyway.
Maybe the limitation should be a bit more obvious, like "You still remember the terms of your contract! You cannot give away your soul to third parties."

Reply:
But without the brainloss here, having your soulreturned won't cause the dumb condition, which is the point of the damage here, preventing infinite contract re-signing.

Reply:
This is a balance change that wasn't addressed